docs/02-connect/01-get_a_wallet.md

```diff
@@ -4,14 +4,398 @@ title: Get a Wallet
 
 
 
-In order to interact with smart contracts in the EVM, you need a valid EVM-compatible wallet such as [MetaMask](https://metamask.io/). The wallet needs to connect to the EON network.
+To interact with smart contracts in the EVM, you need a valid EVM-compatible wallet. Use one of the following wallets to connect to the EON network:
 
+* [Cobalt](https://chrome.google.com/webstore/detail/cobalt/hekbjgfncacdinlajhgiakpaieajpfph/related)
+* [MetaMask](https://metamask.io/)
 
-### Install MetaMask
+## Using Cobalt Wallet
 
-**Note:** The MetaMask connection is illustrated using the Chrome browser.
+Cobalt is a web browser extension wallet available using Chrome or Brave. Like a physical wallet, Cobalt holds ZEN cryptocurrency and tokens. Connecting Cobalt to the EON network (Gobi Testnet) allows you to interact with the EON sidechain (Horizen’s EVM) and dApps.
+
+Cobalt allows you to create and restore wallets from any EVM-compatible network. Once you have configured the network in Cobalt, you can set a link to the corresponding block explorer to view your account transaction history. 
+
+### Before You Begin
+
+In this discussion, the [Chrome web browser](https://chrome.google.com/webstore/detail/cobalt/hekbjgfncacdinlajhgiakpaieajpfph/related) is used to illustrate the instructions to install and use the Cobalt wallet. 
+
+In a web browser, open the Cobalt web extension in the Chrome Web Store. Add the Cobalt extension. Then, click the Cobalt icon in the top-right corner of your browser.  The Cobalt Welcome! page appears.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-welcome.png" alt="Cobalt welcome" width="300" height="200" />
+</p>
+
+### Create an Account
+To begin, create an account for your wallet. Make sure that your password meets the following criteria:
+
+* Your password must contain at least 16-characters.
+* No common passwords are accepted, such as password, 12345, etc.
+* Your password cannot be saved to a keychain, therefore keep your password in a safe place.
+
+Perform the following steps:
+
+1. Enter a **password**. 
+2. Enter your password again to **confirm**.
+3. Click **Terms of Use**. After reading the terms and conditions, click the **checkbox** to agree on the terms.
+4. Click **Create**.
+
+
+### Create a New Wallet
+
+1. To create the new wallet, click **Create Wallet**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-getstarted.png" alt="Cobalt 2-sidechains" width="300" height="200" />
+</p>
+
+2. The **Choose a wallet** page appears. Click **Horizen EON** or **TokenMint** to create a wallet on a supported sidechain. 

```

Comment:
We should direct the user to click "Horizen EON" bc this is EON documentation.
Something like:
```suggestion
2. The **Choose a wallet** page appears. There are options to create a wallet for supported sidechains **Horizen EON** or **TokenMint**. Click **Horizen EON**.
The wallet created for Horizen EON is compatible with any other EVM-compatible chain. 

```